src/rate_limiter.rs

```diff
@@ -182,45 +188,51 @@ mod tests {
     use tiny_keccak::sha3_256;
     use types::MessageId;
 
-    /// Creates a random `GetIData` request and returns it serialised along with its hash digest.
-    fn create_get_idata_request() -> (Vec<u8>, Digest256) {
-        let payload = unwrap!(serialisation::serialise(
-            &UserMessage::Request(Request::GetIData {
-                name: rand::random(),
-                msg_id: MessageId::new(),
-            }),
-        ));
-        let hash = sha3_256(&payload);
-        (payload, hash)
-    }
+    static LARGE_MESSAGE: [u8; CLIENT_CAPACITY as usize] = [0; CLIENT_CAPACITY as usize];
```

Comment:
I was just trying to save resources slightly and avoid having a bunch of duplicated large arrays (which I think would happen if we used `const` instead, right?)